### ISSUE TITLE
Is this also useful for local images?

### ISSUE THREAD

**Issue by cristianoccazinsp** [DylanVann/react-native-fast-image#469]
Just wondering, does this provide any significant performance boost when loading images from a local source? Trying to build a gallery viewer, but react is just way too slow to render images with <Image /> takes ~1 second to load just 10-15 images in a FlatList.

**Comment by DylanVann** [comment]
I don’t think it would make much of a difference if you use this vs. Image for local images.

What you need to do is make sure you’re loading thumbnails of the right size, if you’re trying to render a grid using the original photos that will probably be super slow and use a lot of memory.

**Comment by cristianoccazinsp** [comment]
That's interesting. Testing with full images it doesn't really seem like React's Image loads it all into memory, I would otherwise be getting lots of out of memory errors. Are you sure there's really any difference between loading the full image vs loading a reduced one?
